app/src/main/java/swati4star/createpdf/util/CreatePdf.java

```diff
@@ -144,11 +142,6 @@ protected String doInBackground(String... params) {
 
                 Log.v("Stage 5", "Image compressed " + qualityMod);
 
-                BitmapFactory.Options bmOptions = new BitmapFactory.Options();
-                Bitmap bitmap = BitmapFactory.decodeFile(mImagesUri.get(i), bmOptions);
-
-                Log.v("Stage 6", "Image path adding");
-
```

Comment:
are you sure we don't need this section?

Reply:
Android studio showed me the bitmap variable wasn't used and after deleting that the bmOptions variable also was marked as not used

Reply:
I understand. But, I remember having some issues with functionality after removing this section. 

Can we leave it as is?
The removal of other files is good

Reply:
ok